### PR TITLE
added  Discord bots  to README.md

### PR DESCRIPTION
**Pull request by alainseys**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [ ] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [ ] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [X] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by kokomo123 — CHANGES_REQUESTED** [review]
Typoed a bit there, DiscordBotPanel, Discord Bot Panel Platform, then the .

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Two links leading to the same place is probably unnecessary. The first link should link to the site to download the tool. 

If there is no link, then the first hyperlink  `[DisocrdBotPanel(https://github.com/jareer12/DiscordBotPanel)` could be the source code, and then no source code link would be needed.

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -969,6 +969,8 @@ Games, game servers and control panels.
 - [A Dark Room](https://github.com/doublespeakgames/adarkroom) - Minimalist text adventure game for your browser. ([Demo](https://adarkroom.doublespeakgames.com/)) `MPL-2.0` `HTML5`
 - [elevatorsaga](https://play.elevatorsaga.com/) - The elevator programming game. ([Source Code](https://github.com/magwo/elevatorsaga)) `MIT` `Javascript`
 - [EmuLinkerSF](https://emulinker.org) - EmuLinkerSF is an open source Kaillera server. Kaillera is a client/server system that any emulator can implement to enable netplay over the Internet. ([Source Code](https://github.com/God-Weapon/EmuLinkerSF)) `GPL-2.0` `Java`
+- [DisocrdBotPanel](https://github.com/jareer12/DiscordBotPanel) - DiscordBotPanelplatform to manage your Discord bots . ([Source Code](https://github.com/jareer12/DiscordBotPanel
```
[the remark this answers is not in the document]
Comment (nodiscc) [reply README.md:972]: - please remove the extra space before `.`

**Comment by jareer12** [comment]
Thanks to everyone for helping.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nodiscc) [reply README.md:972]: @alainseys ping

**Comment by nodiscc** [comment]
No reply for a month, closing.

**Pull request closed without merging**